pkg/kubeclient/doc.go

```diff
@@ -0,0 +1,56 @@
+/*
+Package kubeclient provides Kubernetes client implementation.
+
+Client implements methods to perform CRUD operations on Kubernetes objects in a cluster.
+
+Create, Get, Update or Delete Kubernetes Objects:
+
+	kc, err := kubeclient.FromConfigOrCluster("/home/dir/config/.kubeconfig")
+	...
+	pod := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "",
+			Kind:       "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "MyPod",
+			Namespace: "ns-1",
+		},
+		Spec: {
+			.....
+			.....
+		}
+	}
+	ctx := context.Background()
+	....
+	err := kc.Get(ctx, pod)
+	...
+	err := kc.Create(ctx, pod)
+	...
+
+	pod.Spec.Image = ....
+	err := kc.Update(ctx, pod)
+	...
+
+
+List Kubernetes Objects:
+	ctx := context.Background()
```

Comment:
should these be examples?

Reply:
I have pkg level example and duplicated in doc.go as well. I will take it out from doc.go after seeing how godocs looks for it.